.gitignore

```diff
@@ -6,3 +6,6 @@ build*
 *.bak
 src/mlpack/core/util/gitversion.hpp
 src/mlpack/core/util/arma_config.hpp
+
+.idea
```

Comment:
No need, I already rolled it back in a8b5be9.  I also made a few simple style changes in c6477da.